<header class="banner"><div class="banner">
  <div class="logos">
    <div><img title="Julia logo" src="../figures/julia-logo-color.svg" alt="Julia animated logo" width=35 /></div>
    <div>◌</div>
    <div><img title="Powered by MathJax" src="https://www.mathjax.org/badge/mj-logo.svg"
              alt="Powered by MathJax" width=65 />
    </div>
  </div>  <!-- logos -->
  <div class="cite_textbook ms9">4.067 <span class="authors"></span></div>
</div></header>

<footer class="navbar navbar-default navbar-fixed-bottom mcs-psy-nav" >
  <div class="container-fluid text-right h4">
    <em>ψᵟ</em>
  </div>
</footer>






### problem 4.067
Figure P4.67 provides steady-state data for a throttling valve in series with a heat exchanger. Saturated liquid Refrigerant 134a enters the valve at a pressure of 9 bar and is throttled to a pressure of 2 bar. The refrigerant then enters the heat exchanger, exiting at a temperature of 10°C with no significant decrease in pressure. In a separate stream, liquid water at 1 bar enters the heat exchanger at a temperature of 25°C with a mass flow rate of 2 kg/s and exits at 1 bar as liquid at a temperature of 15°C. Stray heat transfer and kinetic and potential energy effects can be ignored. 



### Given

![4.067 diagram](../figures/EAT_14995565987270_24363356200136077.png)

### Find
  $\quad$(a) the temperature, in °C, of the refrigerant at the exit of the valve, and   
  $\quad$(b) the mass flow rate of the refrigerant, in kg/s.   




### Get Organized   

List your assumptions:   
1. Steady state operation  
2. No heat losses from any components  
3. No kinetic and potential energy effects.

Let's determine what information we need to find in order to answer the questions.  

#### (a) volumetric flow rate of the steam at the inlet to the first-stage turbine, 

$\quad$ We need $\dot V_1.$ We are given $\dot m.$ We know $\dot V_1 = \dot m v_1$   
$\quad$ We'll resolve State ⓵ to get $v_1.$


#### (b) the rate of heat transfer to the steam flowing through the reheater   
$\quad$ We apply the energy equation to the reheater. _Remember, the steam is flowing, so it carries internal energy and flow energy in/out of the system._  

\begin{equation*}
 \dot E_{in} - \dot E_{out} = \color{red}\cancelto{\text{0: steady state}}{\frac {\rm d}{\rm dt}E_{sys}} \\[6pt] 
 \dot Q_\mathit{reheater,in} + \dot m \left( u_2 + P_2v_2 \right) - \dot m \left( u_3 + P_3v_3 \right) = 0 \\[12pt]
 \dot Q_\mathit{reheater,in} = \dot m \left( h_3 - h_2 \right)   
\end{equation*}

$\quad$ We'll resolve States ⓶ and ⓷ to get $h_3, h_2.$  


#### (c) the total power produced by the two stages of the turbine   
$\quad$ We apply the energy equation to each turbine and sum the results.  

$$\dot W_{total,out} = \dot W_\textit{first stage, out} + \dot W_\textit{second stage, out}$$  

$\quad$ We'll start with first stage turbine. _Again, the steam is flowing, so it carries internal energy and flow energy in/out of the system._   

\begin{equation*}
 \dot E_{in} - \dot E_{out} = \color{red}\cancelto{\text{0: steady state}}{\frac {\rm d}{\rm dt}E_{sys}} \\[6pt] 
 \dot m \left( u_1 + P_1v_1 \right) - \dot W_\textit{first stage, out} - \dot m \left( u_2 + P_2v_2 \right) = 0 \\[12pt]
 \dot W_\textit{first stage, out} = \dot m \left( h_1 - h_2 \right)   
\end{equation*}

$\quad$ Likewise, 
\begin{equation*}
 \dot W_\textit{second stage, out} = \dot m \left( h_3 - h_4 \right)  \\[15pt]
\end{equation*}

$\quad$ Finally, 
\begin{align*}
 \dot W_{total,out} 
     &= \dot m \left( h_1 - h_2 \right) + \dot m \left( h_3 - h_4 \right) \\ \\
\end{align*}  

$\quad$ Overall, the properties we need are $v_1,\ h_1,\ h_2,\ h_3,\ h_4.$

### Attack 

 Today, we'll use Julia, with _Unitful.jl_, a package for calculation with physical units.

In [1]:
using Unitful       # Unitful units package
const kJ = u"kJ"    # make shortcuts to Unitful units constructor
const kg = u"kg"
const m = u"m"
const kW = u"kW"
const minute = u"minute"
const hr = u"hr";

#### (a) volumetric flow rate of the steam at the inlet to the first-stage turbine, 

$\quad \dot V_1 = \dot m v_1$   

$\quad \left.
\begin{alignat*}{1}
P_1 &= 40\ \textsf{bar} \\
T_1 &= 500\ \textsf{℃} 
\end{alignat*}
\right\}
\rightarrow
v_1 = 0.08643\, \mathrm{m^3/kg}, \quad h_1 = 3445.3\, \mathrm{kJ/kg}
$  

$\quad$ _We get $h_1$ while we're in the table, knowing we need it later._

In [2]:
ṁ = 40000kg/hr              # given
h₁ = 3445.3kJ/kg            # table 
v₁ = 0.08643m^3/kg          # table

V̇₁ = ṁ * v₁ |> m^3/minute

57.620000000000005 m^3 minute^-1

$\color{red} \dot V_1 = 57.62\ \rm m^3/min$

<p style = " background-image:  url(./figures/bacgroundTiles.png); line-height: 0.35em" >&nbsp;</p>

#### (b) the rate of heat transfer to the steam flowing through the reheater   
$\quad \dot Q_\mathit{reheater,in} = \dot m \left( h_3 - h_2 \right) $

$\quad \left.
\begin{alignat*}{1}
P_3 &= 20\ \textsf{bar} \\
T_3 &= 500\ \textsf{℃} 
\end{alignat*}
\right\}
\rightarrow
 h_3 = 3467.6\, \mathrm{kJ/kg}
$
$\quad \left.
\begin{alignat*}{1}
P_2 &= 20\ \textsf{bar} \\
T_2 &= 400\ \textsf{℃} 
\end{alignat*}
\right\}
\rightarrow
 h_2 = 3247.6\, \mathrm{kJ/kg}
$ 

In [3]:
h₃ = 3467.6kJ/kg            # table 
h₂ = 3247.6kJ/kg            # table 

Q̇_reheater_in = ṁ * (h₃ - h₂) |> kW

2444.4444444444443 kW

$\color{red} \dot Q_\mathit{reheater,in} = 2444.44\, \rm kW$

<p style = " background-image:  url(./figures/bacgroundTiles.png); line-height: 0.35em" >&nbsp;</p>

#### (c) the total power produced by the two stages of the turbine   
$\quad \dot W_{total,out} = \dot m \left( h_1 - h_2 \right) + \dot m \left( h_3 - h_4 \right) $

$\quad \left.
\begin{array}{l}
P_4 = 0.6\ \textsf{bar} \\
\text{saturated vapor}
\end{array}
\right\}
\rightarrow
 h_4 = 2653.5\, \mathrm{kJ/kg}
$

In [4]:
h₄ = 2653.5kJ/kg            # table 

Ẇ_total_out = ṁ * (h₁ - h₂ + h₃ - h₄) |> kW

11242.222222222224 kW

$\color{red} \dot W_{total,out} = 11242.22\ \rm kW$

<footer style="background-color: #461d7c; color:rgba(253,208,35,0.35); line-height:2em; padding-left:1em; padding-right:0.5em; bottom:0; float:bottom; text-align:right;"><i>ψᵟ</i></footer>